app/views/application/_header_product.html.erb

```diff
@@ -4,7 +4,7 @@
       <%= button_to edit_vendor_product_path(@product.vendor, @product), :method => :get, :class => "btn btn-default" do %>
         <i class="fa fa-fw fa-wrench" aria-hidden="true"></i> Edit Product
       <% end %>
-      <% if mode_atl?%>
+      <% if current_user.user_role?([:atl, :admin]) %>
```

Comment:
It wouldn't have fired in Demo mode before, since it would only fire in ATL mode. We've been trying to move away from using the modes for permissions and use the user_role? method so that it always works in precert mode. I suppose we could add demo mode here though...